doc/source/user_guide_embedding/libraries.rst

```diff
@@ -0,0 +1,34 @@
+.. _ref_embedding_user_guide_libraries:
+
+*********
+Libraries
+*********
+
+
+.. note::
+
+    This is an experimental feature. Some of these libraries will not work.
+
+Most of Mechanical's scripting APIs are implemented in C#. However, there are a small number
+of Python modules that are distributed with the installation of Mechanical that can be used
+from within the Mechanical Scripting View. These modules are not available for use from an
```

Comment:
Mechanical Scripting pane.

Reply:
@sbradrick I reworded this using similar language in https://github.com/ansys/pymechanical/pull/412